tqdm/_utils.py

```diff
@@ -19,7 +19,7 @@
 try:  # pragma: no cover
     import colorama
     colorama.init()
-except ImportError:  # pragma: no cover
+except:  # pragma: no cover
```

Comment:
After discussion in #126, I guess this is the fix you mentioned @casperdcl. Ok I agree with the change, but I think that we should print a warning if the exception is different than ImportError to be easier for us to debug in the future.